### PR TITLE
Reformatting of quizzes

### PR DESCRIPTION
**Pull request by craiga**
Updating quizzes so the numbering on the screen matches the text.

Also reformatting so they're a little more pleasant.

### CODE REVIEW

File: learning/web/content/modules/benzodiazepines/CON234573_21.mdx [hunk 1]
```diff
@@ -15,15 +15,14 @@ import Expander from '../../../src/components/Expander'
...
-* If i, ii and iii are correct
-* If ii and iv are correct
-* If only v is correct
-* If i, ii, iii and v are correct
+* If 1, 2 and 3 are correct
+* If 2 and 4 are correct
+* If only 5 is correct
+* If 1, 2, 3 and 5 are correct
 * If all are correct
 [**Feedback on the responses is given at the end of the module**](/benzodiazepines/CON234573_22#q1)
```
Comment (DimplePatel) [inline learning/web/content/modules/benzodiazepines/CON234573_21.mdx]: is there a missing line break here?

File: learning/web/content/modules/benzodiazepines/CON234573_21.mdx [hunk 2]
```diff
@@ -46,15 +44,14 @@ import Expander from '../../../src/components/Expander'
 * Substitute the benzodiazepine for another class of anxiolytics
 **Choose your answers as follows:**
```
Comment (DimplePatel) [inline learning/web/content/modules/benzodiazepines/CON234573_21.mdx]: is there a missing line break here?

File: learning/web/content/modules/benzodiazepines/CON234573_21.mdx [hunk 3]
```diff
@@ -65,15 +62,14 @@ import Expander from '../../../src/components/Expander'
...
-* If i, ii and iii are correct
-* If ii and iv are correct
-* If only v is correct
-* If i, ii, iii and v are correct
+* If 1, 2 and 3 are correct
+* If 2 and 4 are correct
+* If only 5 is correct
+* If 1, 2, 3 and 5 are correct
 * If all are correct
 [**Feedback on the responses is given at the end of the module**](/benzodiazepines/CON234573_22#q4)
```
Comment (DimplePatel) [inline learning/web/content/modules/benzodiazepines/CON234573_21.mdx]: is there a missing line break here?

File: learning/web/content/modules/benzodiazepines/CON234573_21.mdx [hunk 4]
```diff
@@ -84,8 +80,7 @@ import Expander from '../../../src/components/Expander'
 * Telephone the UK National Poisons Information Service for guidance because poisoning in a child needs to be taken very seriously
 [**Feedback on the responses is given at the end of the module**](/benzodiazepines/CON234573_22#q5)
```
Comment (DimplePatel) [inline learning/web/content/modules/benzodiazepines/CON234573_21.mdx]: is there a missing line break here?

File: learning/web/content/modules/benzodiazepines/CON234573_21.mdx [hunk 5]
```diff
@@ -96,15 +91,14 @@ import Expander from '../../../src/components/Expander'
...
-* If i, ii and iii are correct
-* If ii and iv are correct
-* If only v is correct
-* If i, ii, iii and v are correct
+* If 1, 2 and 3 are correct
+* If 2 and 4 are correct
+* If only 5 is correct
+* If 1, 2, 3 and 5 are correct
 * If all are correct
 [**Feedback on the responses is given at the end of the module**](/benzodiazepines/CON234573_22#q6)
```
Comment (DimplePatel) [inline learning/web/content/modules/benzodiazepines/CON234573_21.mdx]: is there a missing line break here?

File: learning/web/content/modules/benzodiazepines/CON234573_21.mdx [hunk 6]
```diff
@@ -115,8 +109,7 @@ import Expander from '../../../src/components/Expander'
 * Torsade de pointes is an important feature of benzodiazepine overdose
 [**Feedback on the responses is given at the end of the module**](/benzodiazepines/CON234573_22#q7)
```
Comment (DimplePatel) [inline learning/web/content/modules/benzodiazepines/CON234573_21.mdx]: is there a missing line break here?

File: learning/web/content/modules/benzodiazepines/CON234573_21.mdx [hunk 7]
```diff
@@ -127,8 +120,7 @@ import Expander from '../../../src/components/Expander'
 * Lormetazepam 1 mg at night in a 56-year-old man whose sleep is disturbed by apnoeic episodes
 [**Feedback on the responses is given at the end of the module**](/benzodiazepines/CON234573_22#q8)
```
Comment (DimplePatel) [inline learning/web/content/modules/benzodiazepines/CON234573_21.mdx]: is there a missing line break here?

File: learning/web/content/modules/benzodiazepines/CON234573_22.mdx [hunk 8]
```diff
@@ -17,11 +17,9 @@ import Expander from '../../../src/components/Expander'
 * **It is alright to drink alcohol socially with this medicine**
 **Choose your answers as follows:**
```
Comment (DimplePatel) [inline learning/web/content/modules/benzodiazepines/CON234573_22.mdx]: is there a missing line break here?

File: learning/web/content/modules/benzodiazepines/CON234573_22.mdx [hunk 9]
```diff
@@ -222,7 +216,7 @@ import Expander from '../../../src/components/Expander'
 * Substitute the benzodiazepine for another class of anxiolytics
 **Choose your answers as follows:**
```
Comment (DimplePatel) [inline learning/web/content/modules/benzodiazepines/CON234573_22.mdx]: is there a missing line break here?

File: learning/web/content/modules/benzodiazepines/CON234573_22.mdx [hunk 10]
```diff
@@ -343,7 +334,7 @@ import Expander from '../../../src/components/Expander'
 * A single dose of a benzodiazepine can interfere with recall of events that happen while clinically significant amount remains in the body
 **Choose your answers as follows:**
```
Comment (DimplePatel) [inline learning/web/content/modules/benzodiazepines/CON234573_22.mdx]: is there a missing line break here?

File: learning/web/content/modules/benzodiazepines/CON234573_22.mdx [hunk 11]
```diff
@@ -546,11 +531,9 @@ import Expander from '../../../src/components/Expander'
 * Beta-lactam allergy
 **Choose your answers as follows:**
```
Comment (DimplePatel) [inline learning/web/content/modules/benzodiazepines/CON234573_22.mdx]: is there a missing line break here?

File: learning/web/content/modules/benzodiazepines/CON234573_21.mdx [hunk 12]
```diff
@@ -15,15 +15,14 @@ import Expander from '../../../src/components/Expander'
 * It is alright to drink alcohol socially with this medicine
 **Choose your answers as follows:**
```
Comment (DimplePatel) [inline learning/web/content/modules/benzodiazepines/CON234573_21.mdx]: is there a missing line break here?

File: learning/web/content/modules/benzodiazepines/CON234573_21.mdx [hunk 13]
```diff
@@ -65,15 +62,14 @@ import Expander from '../../../src/components/Expander'
 * A single dose of a benzodiazepine can interfere with recall of events that happen while clinically significant amount remains in the body
 **Choose your answers as follows:**
```
Comment (DimplePatel) [inline learning/web/content/modules/benzodiazepines/CON234573_21.mdx]: is there a missing line break here?

File: learning/web/content/modules/benzodiazepines/CON234573_21.mdx [hunk 14]
```diff
@@ -96,15 +91,14 @@ import Expander from '../../../src/components/Expander'
 * Beta-lactam allergy
 **Choose your answers as follows:**
```
Comment (DimplePatel) [inline learning/web/content/modules/benzodiazepines/CON234573_21.mdx]: is there a missing line break here?

File: learning/web/content/modules/benzodiazepines/CON234573_21.mdx [hunk 15]
```diff
@@ -34,8 +33,7 @@ import Expander from '../../../src/components/Expander'
 * Withdrawal effects typically occur only if benzodiazepine treatment lasts longer than several months
 [**Feedback on the responses is given at the end of the module**](/benzodiazepines/CON234573_22#q2)
```
Comment (DimplePatel) [inline learning/web/content/modules/benzodiazepines/CON234573_21.mdx]: is there a missing line break here?

File: learning/web/content/modules/benzodiazepines/CON234573_21.mdx (hunk 1 above)
Comment (craiga) [reply learning/web/content/modules/benzodiazepines/CON234573_21.mdx]: Yes; I haven't got up to Benzodiazepines yet. I just did a search and replace on the changed phrases above.

File: learning/web/content/modules/benzodiazepines/CON234573_21.mdx (hunk 2 above)
Comment (craiga) [reply learning/web/content/modules/benzodiazepines/CON234573_21.mdx]: Yes; I haven't got up to Benzodiazepines yet.

**Pull request merged**